frontend/src/store/auth/reducer.ts

```diff
@@ -1,13 +1,16 @@
-import { createReducer } from '@reduxjs/toolkit';
+import { createReducer, isAnyOf } from '@reduxjs/toolkit';
 import { DataStatus } from 'common/enums/enums';
-import { signUp } from './actions';
+import { signUp, signIn, loadCurrentUser } from './actions';
 
 type State = {
   dataStatus: DataStatus;
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  user: any; // will be change
```

Comment:
Maybe {}, because backend return {user: {email: email, id: id}, token: token}?
```suggestion
  user: {};
```